### PR TITLE
Apply collection operation, clear on import, description on csv import/export

### PR DESCRIPTION
**Pull request by tsilva**

### CODE REVIEW

File: src/budy/models/product.py [hunk 1]
```diff
@@ -294,6 +300,18 @@ def simple_csv_url(cls, absolute = False):
             absolute = absolute
         )
 
+    @appier.operation(
+        name = "Apply Collection",
+        parameters = (
+            ("Collection", "collection", "str"),
+        )
+    )
+    def apply_collection(self, collection):
```
Comment (tsilva) [inline src/budy/models/product.py]: This is failing saying that method is expecting two arguments but only one is provided. I found no examples of non class method operations that accepted arguments in the entire workspace.

Comment (joamag) [inline src/budy/models/product.py]: any special reason to use the `add` method instead of `append` ?

Comment (tsilva) [inline src/budy/models/product.py]: no, should be `append`. fixed.

Comment (joamag) [inline src/budy/models/product.py]: If the method saves something it should end with `_s`

**Pull request merged**